resources/js/pages/game/[game].tsx

```diff
@@ -58,6 +60,14 @@ const GameShow: AppPage = () => {
         ogImage={backingGame!.badgeUrl}
       />
 
+      {/* TODO desktop preload image */}
+      {ziggy.device === 'mobile' && (game.banner?.mobileSmAvif || game.imageIngameUrl) ? (
+        <SEOPreloadImage
+          src={game.banner?.mobileSmAvif ?? (game.imageIngameUrl as string)}
+          type={game.banner?.mobileSmAvif ? 'image/avif' : 'image/png'}
+        />
```

Comment:
To load the banner image as fast as possible, we communicate all sorts of hints to the browser that it needs to prioritize it.